GOVERNANCE.md

```diff
@@ -1,39 +1,55 @@
 # Cloud Native Buildpacks Governance
-This document defines the governance structure for Cloud Native Buildpacks. This project is committed to building an open, inclusive, productive and self-governing open source community that maintains a specification and high-quality tools for translating source code into OCI images. The guidelines herein describe how the Cloud Native Buildpacks community should work together to achieve this goal.
+This document defines the governance structure for Cloud Native Buildpacks. This project is committed to building an open, inclusive, productive and self-governing open source community that maintains a specification and high-quality tools for translating source code into OCI images. The guidelines herein describe how the Cloud Native Buildpacks (CNB) community should work together to achieve this goal.
 
 ## Repositories
-The following code repositories are governed by the Cloud Native Buildpacks community and maintained under the [buildpacks](https://github.com/buildpacks) organization.
+All repositories under the [buildpacks](https://github.com/buildpacks) organization are maintained and governed by the buildpacks project.
 
-* [spec](https://github.com/buildpacks/spec)
-* [lifecycle](https://github.com/buildpacks/lifecycle)
-* [pack](https://github.com/buildpacks/pack)
-* [imgutil](https://github.com/buildpacks/imgutil)
-* [docs](https://github.com/buildpacks/docs)
-* [rfcs](https://github.com/buildpacks/rfcs)
-* [resources](https://github.com/buildpacks/resources)
-* [community](https://github.com/buildpacks/community)
-* [samples](https://github.com/buildpacks/samples)
+## [Technical Oversight Committee](TEAMS.md#Technical-Oversight-Committee)
 
-## [Core Team](TEAMS.md#Core-Team)
-The Core team is responsible for the direction of the project (roadmap), subteam leadership, the spec, and cross-cutting concerns. This includes:
+The CNB Technical Oversight Committee (TOC) is modeled on the [CNCF TOC](https://github.com/cncf/toc) as a technical governing body. It oversees all aspects of the project and has a mandate to drive consensus for:
 
-* [spec](https://github.com/buildpacks/spec)
-* [rfcs](https://github.com/buildpacks/rfcs)
+* defining and maintaining the technical vision for the CNB project.
+* fostering a healthy and welcoming community, including by defining and enforcing our [Code of Conduct](https://github.com/buildpacks/.github/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
I would say yes, because
1. It's the only oversight committee that exists and fulfills other non-technical steering-like responsibilities
2. It is responsible for personnel decisions like appointing the team leads and creating a welcoming and inclusive community should be a priority when appointing folks to leadership positions
3. We gave the TOC responsibility for governance and project priorities (via the roadmap) and this is related to both governance and priorities.
4. We don't say that the TOC is solely responsible for any of these items but "a mandate to drive consensus for..."

It is everyone's responsibility to foster a welcoming community but the TOC should set the example and own the code of conduct document as it does the governance structure.

I would be happy to say it is the combined responsibility of the team leads and the TOC. I thought about adding a Leadership team section that defines leadership team as the union of TOC and leads and assigns shared responsibilities.

Reply:
Rename to "Oversight Committee"? ;)

Reply:
For context, I was talking to Joe about this and I wanted to empower Team Leads to be able to do this in the community. If this is just part of "governance", then I'm fine with it as is.